compact_proof/docker/ceremony.Dockerfile

```diff
@@ -1,4 +1,34 @@
 # syntax=docker/dockerfile:1
+FROM rust:1.76.0 AS dependencies
```

Comment:
Let's stick to using the same version that is in use in `risc0/risc0/rust-toolchain.toml`